Branch-SDK/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     android:versionCode="1"
     android:versionName="1.0" >
+    <uses-permission android:name="android.permission.READ_LOGS"
+        tools:ignore="ProtectedPermissions" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Since this is a shipped manifest, and needed, these should be obtained at runtime. But if I'm reading this PR correctly, these shouldn't be required since you can just grab the logs we generate directly.

Reply:
Removed, thanks @gdeluna-branch !